frontend/.env.development

```diff
@@ -1,8 +1,6 @@
-REACT_APP_BACKEND_HOST=https://localhost:8443
+REACT_APP_BACKEND_HOST=http://localhost:8080
 REACT_APP_DEBUG=ON
 REACT_APP_GITHUB_CLIENT_ID=<put your github app id here>
```

Comment:
Are you sure it's right that everyone needs their own oauth application? Can we not just have one oauth application for everything?

Reply:
Yes, github themselves say here (https://developer.github.com/v3/guides/basics-of-authentication) not to check in that client secret. 

Basically if I were to create a secret and just check it in then everyone uses my id and secret while doing dev. Suppose someone did something dodgy that violated githubs terms with that authenticated app and they decided to take action, I would be the one held responsible so a ban or something would be possible.

I've never seen a case for oauth login apps where users use a shared id and secret. Every dev always just makes their own.

Reply:
I get that we shouldn't commit the secret, that makes sense. Side-note, using environment variables sounds like a good idea as per that page's suggestion so we don't have git changes for it.
How is ng going to work though? Because we're saying that we need an oauth app for ng to use, which means the secret is going to have to be in a shared place (eg env variable on the ng box), and all users will use that same oauth app in that case right?

Reply:
We can only user environment variables for the backend though. You will still need to use the .env file for the frontend. I'm not too fussed either way for the backend but I left if using the application variables so its easier to debug and change.

For ng we can get an organisation oauth app set up as per here https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/. Basically just have to get steve onto it as he has admin access to the orion health org.

Reply:
Ok, my question though is - is there a reason we can't use the organisation oauth app for dev as well?

Reply:
We could we just shouldn't commit it. Giving out the org one to all devs is probably a bit dodgy. Being as though all of us work for orion its fine as we will have access to the server it's stored on, but suppose someone outside our team wanted to use it, then they would make their own.

This being said as it takes 2 minutes to create a new one its easier to just tell everyone to create a new one.

Reply:
Yep ok. Yeah I know it's easy to create one, I was just seeing if we could minimise the number of steps that a new dev would have to go through to get up and running. If you're familiar with oauth it's easy but if not then people might get confused.
FYI I'm still thinking a mock login component just for dev could be useful, I might have a go at building one since it should be fairly easy. That would allow new devs to not have to worry about github oauth unless they really wanted to (ie an optional step).

Reply:
I was talking to Steve this morning and he said he still wants basic auth so maybe that could be the easier way for people to set it up. We will either need a sign up sort of functionality or people will need to insert stuff directly into mongo.